Source/JavaScriptCore/runtime/Options.cpp

```diff
@@ -771,7 +771,8 @@ void Options::notifyOptionsChanged()
     if (!Options::allowDoubleShape())
         Options::useJIT() = false; // We don't support JIT with !allowDoubleShape. So disable it.
 
-    if (!Options::useWasm())
+    // When reenabling JITLess wasm we should unskip the tests disabled in https://bugs.webkit.org/show_bug.cgi?id=281857
+    if (!Options::useWasm() || !Options::useJIT())
```

Comment:
Whoa, this condition is invalid since it means `--useWasm=1` is totally ignored. 

Perhaps we should be checking `OptionsHelper::wasOverridden(Options::useWasmID)` as in my [patch](https://github.com/WebKit/WebKit/commit/2c0494fcc3fe018670c408476931c3698d9b2ee2) from last year.